src/core/installAddon.js

```diff
@@ -405,11 +419,25 @@ export class WithInstallHelpers extends React.Component {
   }
 
   enable() {
-    const { _addonManager, dispatch, guid, iconUrl, name } = this.props;
+    const {
+      _addonManager,
+      _tracking,
+      dispatch,
+      guid,
+      iconUrl,
+      name,
+      type,
+    } = this.props;
 
     return _addonManager
       .enable(guid)
       .then(() => {
+        _tracking.sendEvent({
+          action: getAddonTypeForTracking(type),
+          category: getAddonEventCategory(type, ENABLE_ACTION),
```

Comment:
this also comes up when we install a theme since we are still special casing this? is that ok?

Reply:
ah, right. maybe we could add an argument to not send an event in that case.

Reply:
OR maybe we can add enable tracking to the others (extensions and lightweight theme) since it technically does do this on install..? but that might be more involved..

Reply:
I have added a prop to the `enable` helper, which prevents the tracking event to be sent when we are installing an add-on.